tests/testthat/test-loopy-post-estimation-no-tuning.R

```diff
@@ -0,0 +1,210 @@
+      9L, "epanechnikov",       10L,
+      14L, "epanechnikov",       10L,
+      20L, "epanechnikov",       10L
+    )
+
+  # ------------------------------------------------------------------------------
+
+  static_1 <- melodie:::make_static(
+    submodel_wflow,
+    param_info = submodel_wflow |> extract_parameter_set_dials(),
+    metrics = metric_set(rmse, rsq),
+    eval_time = NULL,
+    split_args = rs_args,
+    control = ctrl
+  )
+
+  data_1 <- melodie:::get_data_subsets(submodel_wflow, rs_split, rs_args)
+  static_1 <- melodie:::update_static(static_1, data_1)
+  static_1$y_name <- "outcome"
+
+  submodel_res <- melodie:::loopy(rs_iter, submodel_grid, static_1)
+  expect_true(!is.null(submodel_res$.metrics[[1]]))
+  expect_named(submodel_res, c(".metrics", ".notes", "id", ".predictions"))
+  expect_true(nrow(submodel_res) == 1)
+  expect_equal(
+    nrow(submodel_res$.predictions[[1]]),
+    nrow(data_1$pred$data) * nrow(submodel_grid)
+  )
+})
+
+test_that("verifying loopy, submodels only, post estimation without tuning", {
+  skip_if_not_installed("modeldata")
+  skip_if_not_installed("kknn")
+  skip_if_not_installed("probably")
+  skip_if_not_installed("mgcv")
+
+  load(system.file(
+    "regression_tests",
+    "submodel_only_example.RData",
+    package = "melodie"
+  ))
+
+  ctrl <- tune::control_grid(save_pred = TRUE)
+
+  # ------------------------------------------------------------------------------
+
+  set.seed(1)
+  dat <- modeldata::sim_classification(1000)
+  rs <- vfold_cv(dat)
+
+  rs_split <- rs$splits[[1]]
+  rs_args <- rsample::.get_split_args(rs)
+
+  rs_iter <- melodie:::vec_list_rowwise(rs) |>
+    purrr::pluck(1)
+
+  # ------------------------------------------------------------------------------
+
+  mod <- nearest_neighbor(neighbors = tune(), weight_func = "triangular") |>
+    set_mode("classification")
+
+  submodel_only_wflow <- workflow(class ~ ., mod, cls_est_post)
+
+  submodel_only_grid <- tibble(neighbors = 3:10)
+
+  # ------------------------------------------------------------------------------
+
+  static_1 <- melodie:::make_static(
+    submodel_only_wflow,
+    param_info = submodel_only_wflow |> extract_parameter_set_dials(),
+    metrics = metric_set(accuracy, roc_auc, brier_class),
+    eval_time = NULL,
+    split_args = rs_args,
+    control = ctrl
+  )
+
+  data_1 <- melodie:::get_data_subsets(submodel_only_wflow, rs_split, rs_args)
+  static_1 <- melodie:::update_static(static_1, data_1)
+  static_1$y_name <- "class"
+
+  submodel_only_res <- melodie:::loopy(rs_iter, submodel_only_grid, static_1)
+  expect_true(!is.null(submodel_only_res$.metrics[[1]]))
+  expect_named(submodel_only_res, c(".metrics", ".notes", "id", ".predictions"))
+  expect_true(nrow(submodel_only_res) == 1)
+  expect_equal(
+    nrow(submodel_only_res$.predictions[[1]]),
+    nrow(data_1$pred$data) * nrow(submodel_only_grid)
+  )
+
+  # TODO more tests can be added when calibration method = "none" is implemented
```

Comment:
We need to have a 1:1 unit test here. When tidymodels/tailor#75 we can update this